build.py

```diff
@@ -42,6 +42,8 @@
 use_plugin("python.coverage")
 use_plugin("python.flake8")
 use_plugin("python.frosted")
+use_plugin("python.sphinx")
+use_plugin('filter_resources')
```

Comment:
Does this mean we weren't using filter resources before?! This would mean that pybuilder.__version__ never worked?!